pom.xml

```diff
@@ -1032,7 +1032,7 @@
                                         </goals>
                                     </pluginExecutionFilter>
                                     <action>
-                                        <ignore />
+                                        <ignore/>
```

Comment:
The original formatting is correct. There should be a space before the `/>` for a self-closed XML tag. I'll update https://github.com/airlift/codestyle to match